packages/default/scss/menu/_variables.scss

```diff
@@ -168,16 +168,10 @@ $kendo-menu-popup-padding-y: null !default;
 /// @group menu
 $kendo-menu-popup-border-width: $kendo-popup-border-width !default;
 
-/// The font sizes of the Menu popup.
```

Comment:
Shall we remove the comments here:
/// The line heights used along with $kendo-font-size.
/// @group menu

Reply:
Okay, I thought that we are using this for all kendo-menu-popup-{size}-font-size variables, but I see in the design system that this is not the case- it takes only the variable right after the description. In this case, I think that we should add descriptions for all size variables as well, because currently, the descriptions don't really match the variables, e.g:

<img width="872" alt="Screenshot 2025-06-20 at 9 37 44" src="https://github.com/user-attachments/assets/57f1eabe-3bd9-4cfb-91cd-2b65dc72e1d6" />